### ISSUE TITLE
Song select screen and gameplay.

### ISSUE THREAD

**Issue by AmberrOsu** [ppy/osu#3854]
There are 2 features missing from lazer, the first I've noticed is that doing **``mode=``** followed by the mode such as taiko does not work, it's shows no maps, I have to search difficulty names instead.

The second feature I've noticed is the fading cursor during gameplay and instead of fading away, it stays on screen making gameplay during mainly during taiko and ctb difficult and can do the same to mania if the cursor is over the play field before starting a map.

**Comment by peppy** [comment]
First issue: #3781 
Second issue: #2901 

Please search before making a new issue.
